addons/gopaddle-lite/enable

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
+
+HELM="$SNAP_DATA/bin/helm3 --kubeconfig=$SNAP_DATA/credentials/client.config"
+
+KUBE_VERSION=$($KUBECTL version | grep Server| cut -d '{' -f 2| cut -d ',' -f 2| cut -d ':' -f 2 |cut -d '+' -f 1|cut -d '"' -f 2)
+
+CURRENT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+
+STATIC_VERSION="24"
+if [ "$KUBE_VERSION" -gt $STATIC_VERSION ]
+then 
+   echo "$KUBE_VERSION is not supported"
+fi
+ 
+
+STATICIP=""
+HELM_VERSION="4.2.4"
+STORAGECLASS=""
+ARGS=$(getopt -a --options i:v:s: --longoptions "ip:,version:,storageclass:" -- "$@")
+eval set -- "$ARGS"
+
+while true; do
+  case "$1" in
+    -i|--ip)
+      STATICIP="$2"
+      shift 2;;
+    -v|--version)
+      HELM_VERSION="$2"
+      shift 2;;
+    --)
+      break;;
+  esac
+done
+
+PUBLICIP=true
+if [ $STATICIP ]
+then
+   echo "static IP of the microk8s cluster: $STATICIP"
```

Comment:
Please be consistent with the number of spaces used for indentation.

Reply:
I have corrected the indentation.